INSTALL.rst

```diff
@@ -55,9 +55,79 @@ Finally run the tests to make sure all is setup correctly:
     python setup.py test
 
 
+Installation for developers with Docker
+---------------------------------------
+
+Grab the code from github, you can optionally follow the above steps to create the
+virtual environment, but that is not neccessary – might be helpful if your IDE wants
+to install dependencies for suggestions though.
+
+Then install the test dependencies inside Docker:
+
+.. code-block:: console
+
+    docker-compose -f docker-compose.deps.yml run --rm pip
+
+To run the tests (e.g. unit):
+
+.. code-block:: console
+
+    docker-compose -f docker-compose.test.yml run --rm unit
+
+
+Installation for testing with inspire-next in Docker
+----------------------------------------------------
+
+Follow the directions to create the virtual environment as above and grab the code from github.
```

Comment:
Maybe link to the option? Is the virtualenv really needed? (the section right above says that's not)